### PR TITLE
Release 1.21.0

### PR DESCRIPTION
**Pull request by AlbertDeFusco**

### CODE REVIEW

File: docs/source/building/meta-yaml.rst [hunk 1]
```diff
@@ -597,11 +641,16 @@ need to put the same selector on multiple lines.
      url: http://path/to/unix/source        # [unix]
      md5: 88510902197cba0d1ab4791e0f41a66e  # [unix]
 
+
+NOTE: The fully rendered recipe file will be written back to the directory as ``meta.yaml.rendered``.
```
Comment (msarahan) [inline docs/source/building/meta-yaml.rst]: more specifically, it's in the info/recipe folder.  This folder does not exist if build/include_recipe is False.

**Comment by msarahan** [comment]
LGTM.  Nice work.

File: docs/source/building/meta-yaml.rst (hunk 1 above)
Comment (AlbertDeFusco) [inline docs/source/building/meta-yaml.rst]: I'm still a bit confused. Is ``meta.yaml.rendered`` written back to the build recipe directory or only to the package tar.bz2 file?

Comment (msarahan) [inline docs/source/building/meta-yaml.rst]: It's written into the build directory, but that eventually gets packaged up into the tarball.

Comment (AlbertDeFusco) [inline docs/source/building/meta-yaml.rst]: Ok, I see. I'll update the description in the metadata section

https://github.com/conda/conda-docs/pull/297/commits/748a69d8a4f7467dd7bf68a4b3aca8364ab5bfbb

**Comment by electronwill** [comment]
:+1:

**Pull request merged**